<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>915</width>
    <height>578</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <widget class="QPushButton" name="btnQuit">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>480</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>退出</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="inputUserID">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>70</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="plainText">
     <string>00001</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelUserID">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>70</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>用户ID</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnSubmitUser">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>160</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>登录</string>
    </property>
    <property name="default">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="labelPassword">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>110</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>密码</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="inputPassword">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>110</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="plainText">
     <string>123456</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelLoginStatus">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>20</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Songti SC</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>· 状态：未登录</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnLogout">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>160</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>退出登录</string>
    </property>
    <property name="default">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QTabWidget" name="tabChatView">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>40</y>
      <width>611</width>
      <height>401</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Tab 1</string>
     </attribute>
     <widget class="QListWidget" name="listWidget">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>1</y>
        <width>601</width>
        <height>361</height>
       </rect>
      </property>
      <property name="flow">
       <enum>QListView::TopToBottom</enum>
      </property>
      <property name="sortingEnabled">
       <bool>false</bool>
      </property>
      <item>
       <property name="text">
        <string>LIANGLEYA</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>LIANGLEYA</string>
       </property>
       <property name="textAlignment">
        <set>AlignTrailing|AlignVCenter</set>
       </property>
      </item>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Tab 2</string>
     </attribute>
    </widget>
   </widget>
   <widget class="QScrollArea" name="scrollAreaFriends">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>200</y>
      <width>201</width>
      <height>271</height>
     </rect>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="scrollAreaFriends_Widget">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>199</width>
       <height>269</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <widget class="QWidget" name="verticalLayoutWidget">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>10</y>
        <width>191</width>
        <height>671</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayoutFriends">
       <property name="sizeConstraint">
        <enum>QLayout::SetMaximumSize</enum>
       </property>
       <item>
        <widget class="QLabel" name="labelSelectFriends">
         <property name="font">
          <font>
           <family>Songti SC</family>
          </font>
         </property>
         <property name="text">
          <string>选择一个好友开始新的聊天：</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButtonFriendExample">
         <property name="text">
          <string>好友1: 00002</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </widget>
   <widget class="QPlainTextEdit" name="inputTextMsg">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>450</y>
      <width>491</width>
      <height>61</height>
     </rect>
    </property>
    <property name="plainText">
     <string>testtesttesttesttesttesttesttesttesttesttesttesttesttesttesttesttesttesttesttesttesttesttesttesttesttesttesttesttesttesttesttesttesttest</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnSendMsg">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>470</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>发送消息</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnDelCurrentTab">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>10</y>
      <width>101</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>关闭当前聊天</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>915</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuchat_client">
    <property name="tearOffEnabled">
     <bool>false</bool>
    </property>
    <property name="title">
     <string>Chat客户端</string>
    </property>
   </widget>
   <addaction name="menuchat_client"/>
  </widget>
  <widget class="QStatusBar" name="statusbar">
   <property name="sizeGripEnabled">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>btnQuit</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>115</x>
     <y>512</y>
    </hint>
    <hint type="destinationlabel">
     <x>400</x>
     <y>253</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnLogout</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>on_btnLogout()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>137</x>
     <y>199</y>
    </hint>
    <hint type="destinationlabel">
     <x>401</x>
     <y>436</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnSubmitUser</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>on_btnSubmitUser()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>120</x>
     <y>196</y>
    </hint>
    <hint type="destinationlabel">
     <x>402</x>
     <y>470</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnSendMsg</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>on_btnSendMsg()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>773</x>
     <y>504</y>
    </hint>
    <hint type="destinationlabel">
     <x>243</x>
     <y>368</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnDelCurrentTab</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>on_btnDelCurrentTab()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>798</x>
     <y>48</y>
    </hint>
    <hint type="destinationlabel">
     <x>396</x>
     <y>46</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>on_btnSubmitUser()</slot>
  <slot>on_btnLogout()</slot>
  <slot>on_btnDelCurrentTab()</slot>
  <slot>on_btnSendMsg()</slot>
 </slots>
</ui>
